### DISCUSSION TITLE
三星新機Galaxy A14亮相 !

### DISCUSSION THREAD

**Discussion by Sylvie0601** [1101-GS4719-C/Python-Community-GS4719-C#1095] [General]
我和家裡人從以前到現在幾乎都是在用三星的手機，幾乎每一種手機型號我們都是過，就價格來說，三星的s系列和z系列都不會比蘋果便宜，我認為後來推出的A系列和M系列都很適合小資族使用，就可以花小錢體驗還不錯的功能。
我目前在使用的是S21，我自己當初就很喜歡這一個系列(也有一個原因他的顏色看啦哈哈)，越用就越覺得不會輸蘋果，剛好我的電腦也是windows系統，現在都可以手機和電腦直接做聯合，不一定要都使用蘋果也可以建立自己的生態系。

推一個我覺得我手機超厲害的功能 : 我手機可以當成無線行充將自己手機電量傳輸給另一台手機(隨便機型，只要能無線充電就可以)，功能好處是可以救急，我覺得超讚，目前還在挖掘其他功能中 !
下面新聞是目前新推的A14 : 
新聞網址 : https://tw.news.yahoo.com/%E4%B8%89%E6%98%9F%E6%96%B0%E6%A9%9Fgalaxy-a14%E4%BA%AE%E7%9B%B8-%E5%A4%A7%E8%9E%A2%E5%B9%95-3%E9%8F%A1%E9%A0%AD-%E5%94%AE%E5%83%B9%E5%85%8D7%E5%8D%83%E5%85%83-060512838.html

**Comment by yuxinlin93** [comment]
雖然不知道手機的行情價，但這個價錢有這個品質我是覺得蠻平易近人的，而且記憶卡容量那麼大，鏡頭也不錯，對我這種手機先吃的拍照狂人來說蠻有吸引力的，但是出的這一些顏色我覺得沒有很好看，不知道為甚麼有一種塑膠感
